dataactcore/scripts/setupErrorDB.py

```diff
@@ -1,38 +1,29 @@
+from dataactbroker.app import createApp
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.models import lookups
 from dataactcore.models.errorModels import FileStatus, ErrorType
-from dataactcore.interfaces.db import databaseSession
 
 
 def setupErrorDB():
     """Create error tables from model metadata."""
-    with databaseSession() as sess:
+
+    with createApp().app_context():
+        sess = GlobalDB.db().session
         insertCodes(sess)
         sess.commit()
 
+
 def insertCodes(sess):
     """Insert static data."""
 
     # insert file status types
-
-    statusList = [(1, 'complete', 'File has been processed'),
-        (2, 'header_error', 'The file has errors in the header row'),
-        (3, 'unknown_error', 'An unknown error has occurred with this file'),
-        (4, 'single_row_error', 'CSV file must have a header row and at least one record'),
-        (5, 'job_error', 'Error occurred in job manager'),
-        (6, 'incomplete', 'File has not yet been validated')]
-    for s in statusList:
-        status = FileStatus(file_status_id=s[0], name=s[1], description=s[2])
+    for s in lookups.FILE_STATUS:
+        status = FileStatus(file_status_id=s.id, name=s.name, description=s.desc)
```

Comment:
Yeah, I would have thought so too. I'm sure there's a good explanation for the behavior I'm seeing below, but I'm also fine leaving this as with the explicit column names and frying some other fish.

```python
class JobStatus(Base):
    __tablename__ = "job_status"
    JOB_STATUS_DICT = None

    job_status_id = Column(Integer, primary_key=True)
    name = Column(Text)
    description = Column(Text)
```
```
In [2]: from dataactcore.models.jobTrackerInterface import JobStatus

In [3]: from dataactcore.models.lookups import JOB_STATUS

In [4]: JOB_STATUS
Out[4]: 
[LookupType(id=1, name='waiting', desc='check dependency table'),
 LookupType(id=2, name='ready', desc='can be assigned'),
 LookupType(id=3, name='running', desc='job is currently in progress'),
 LookupType(id=4, name='finished', desc='job is complete'),
 LookupType(id=5, name='invalid', desc='job is invalid'),
 LookupType(id=6, name='failed', desc='job failed to complete')]

In [5]: test = JobStatus(*JOB_STATUS[0])
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-5-d1f5210372ed> in <module>()
----> 1 test = JobStatus(*JOB_STATUS[0])

TypeError: __init__() takes 1 positional argument but 4 were given
```